diff --git a/setup_env b/setup_env
index 35e77d2..840cc7b 100644
--- a/setup_env
+++ b/setup_env
@@ -20,3 +20,4 @@ export CPU2006DATA=${PWD}/benchmarks/spec_cpu2006/data
 # zsim
 export ZSIMPATH=${PWD}/zsim
 export ZSIMHOOKS=${PWD}/zsim/misc/hooks
+export PATH=/bin:$PATH
diff --git a/zsim/src/pin_cmd.cpp b/zsim/src/pin_cmd.cpp
index 817ef02..65f697b 100644
--- a/zsim/src/pin_cmd.cpp
+++ b/zsim/src/pin_cmd.cpp
@@ -50,6 +50,7 @@ PinCmd::PinCmd(Config* conf, const char* configFile, const char* outputDir, uint
     }
 
     args.push_back(pinPath);
+    args.push_back("-ifeellucky");
 
     //Global pin options
     args.push_back("-follow_execv"); //instrument child processes
